build/ci/set-up-mms.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+# Copyright 2021 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+while getopts 'h:' opt; do
+  case ${opt} in
+  h) hostsFile="${OPTARG}" ;; # Output of Evergreen host.list
+  *) exit 1 ;;
+  esac
+done
+
+hosts=$(
+  cat <<EOF | python - "${hostsFile}"
+import sys
+import json
+with open(sys.argv[1]) as hostsfile:
+    hosts = json.load(hostsfile)
+    for host in hosts:
+        print(host["dns_name"])
+EOF
+
+)
+
+cd ..
+cd ..
+
+for host in ${hosts}; do
+
+echo "set service"
+  ./bin/mongocli config set service ops-manager
+
+echo "set ops_manager_url"
+  ./bin/mongocli config set ops_manager_url "http://${host}:9080/"
+done
+
+echo "create first user"
+./bin/mongocli om owner create --firstName evergreen --lastName evergreen --email evergreenTest@gmail.com --password "evergreen1234_" -o json > apikeys.json
+
```

Comment:
This endpoint returns the programmatic keys only for mms 4.4 or later. I am not sure how we can get the programmatic keys for 4.2, maybe we can ask for a backport.

Reply:
it's ok if we don't fully test 4.2, we offer a best effort for 4.2 but as we move to 5.0 we need to keep compatibility with 4.4